libbuteosyncfw/profile/ProfileManager.cpp

```diff
@@ -1137,7 +1149,8 @@ QDomDocument ProfileManagerPrivate::constructProfileDocument(const Profile &aPro
 bool ProfileManagerPrivate::writeProfileFile(const QString &aProfilePath,
         const QDomDocument &aDoc)
 {
-    //FUNCTION_CALL_TRACE;
+    FUNCTION_CALL_TRACE;
+    LOG_WARNING("writeProfileFile() called, forcing disk write:" << aProfilePath);
```

Comment:
@chriadam I could find such WriteOnly means Truncate too implications. Would not make any sense either as it would make it impossible to overwrite existing data and truncating when final size is known - which could be handy in near disk full situations.